vmm/src/config.rs

```diff
@@ -2064,17 +2082,28 @@ pub struct RestoreConfig {
     pub source_url: PathBuf,
     #[serde(default)]
     pub prefault: bool,
+    #[serde(default)]
+    pub net_ids: Option<Vec<String>>,
+    #[serde(default)]
+    pub net_fds: Option<Vec<i32>>,
 }
 
 impl RestoreConfig {
     pub const SYNTAX: &'static str = "Restore from a VM snapshot. \
         \nRestore parameters \"source_url=<source_url>,prefault=on|off\" \
```

Comment:
Also extend this example to include net_ids and net_fds.

Reply:
Added